dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -1083,6 +1105,28 @@ DataInterfaceIIDM::getStaticParameterBoolValue(const std::string& staticID, cons
   return findComponent(staticID)->getStaticParameterValue<bool>(refOrigName);
 }
 
+void
+DataInterfaceIIDM::copy(const DataInterfaceIIDM& other) {
+  networkIIDM_  = other.networkIIDM_;  // No clone here because iidm network is not copyable
+  // Criterias are not copied and must be initialized again
```

Comment:
isn't it possible to reinit the criterias with the local components based on the other data?
Moreover where is this re-creation done?

Reply:
Not a good idea, as the criteria are initialized by the simulation. With this function, we have in output the data interface corresponding to the one that is built by the data interface factory

Reply:
ok, so where do we create again the criteria?